runtime/compiler/runtime/J9CodeCache.cpp

```diff
@@ -134,6 +143,40 @@ J9::CodeCache::initialize(TR::CodeCacheManager *manager,
 
    if (!self()->OMR::CodeCache::initialize(manager, codeCacheSegment, allocatedCodeCacheSizeInBytes))
       return false;
+
+#ifdef LINUX
+   if (manager->isDisclaimEnabled())
+      {
+      uint8_t *middle  = _warmCodeAlloc + (_coldCodeAlloc - _warmCodeAlloc) / 2;
```

Comment:
Should we use `_coldCodeAllocBase` like you used at the moment when disclaim happens?

Reply:
Done.